### PR TITLE
Update README for Rust 1.59 strip feature

### LINKED ISSUE #33: Update strip documentation for upcoming stable Cargo release
- https://github.com/rust-lang/rust/pull/90058
- https://github.com/rust-lang/cargo/pull/10088

Once this PR lands and is released, update minimum Rust version for `strip`  Cargo feature to a stable release.

Also remove reference to `cargo-strip` at this time.

### PR DESCRIPTION
**Pull request by Frago9876543210**
this PR closes #33

### CODE REVIEW

**Comment by Frago9876543210** [comment]
currently `https://doc.rust-lang.org/cargo/reference/profiles.html#strip` doesn't work due to fresh rustc release

**Comment by memoryruins** [comment]
`rustc`'s `-C strip` option is in today's 1.58 release, but `cargo`'s configuration will not be available until the next version, stable `1.59`, since its stabilization (<https://github.com/rust-lang/cargo/pull/10088>) and the sync with rust-lang/rust (<https://github.com/rust-lang/rust/pull/92567>) only merged 9 days ago.

File: README.md [hunk 1]
```diff
@@ -46,16 +39,13 @@ To remove this, run [`strip`](https://linux.die.net/man/1/strip) on the `.elf` f
 $ strip target/release/min-sized-rust
 ```
 
-Available starting `1.45.0-nightly (2020-05-28)`,
-[Cargo has `strip` functionality built in](https://doc.rust-lang.org/cargo/reference/unstable.html#profile-strip-option):
+[Cargo has `strip` functionality built in](https://doc.rust-lang.org/cargo/reference/profiles.html#strip):
 
-![Minimum Rust: Nightly](https://img.shields.io/badge/Minimum%20Rust%20Version-nightly%201.45.0-orange.svg)
+![Minimum Rust: 1.58](https://img.shields.io/badge/Minimum%20Rust%20Version-1.58-brightgreen.svg)
```
Comment (yerke) [inline README.md:44]: I believe this needs to be `1.59`.

**Comment by elias123tre** [comment]
I have created PR #35 that can be merged before release 1.59 (2022-02-24) with info on how to use strip with stable release 1.58 and info about cargo feature in 1.59.

**Comment by Frago9876543210** [comment]
can be merged now

**Comment by johnthagen** [comment]
Blog post: https://blog.rust-lang.org/2022/02/24/Rust-1.59.0.html#creating-stripped-binaries

**Comment by johnthagen** [comment]
Thanks all!

**Pull request merged**